src/tests/containerizer/cgroups2_tests.cpp

```diff
@@ -33,7 +33,29 @@ namespace mesos {
 namespace internal {
 namespace tests {
 
-class Cgroups2Test : public TemporaryDirectoryTest {};
+const string TEST_CGROUP = "test";
+
+// Cgroups2 test fixture that ensures `TEST_CGROUP` does not
+// exist before and after the test is run.
+class Cgroups2Test : public TemporaryDirectoryTest
+{
+public:
+  static void SetUpTestCase()
```

Comment:
this looks like it should be `void SetUp() override` and below `void TearDown() override`, which will run at the start and end of each test, as is, these statics only run at the start and end of the fixture